supermarq-benchmarks/supermarq/benchmarks/phase_code_test.py

```diff
@@ -1,4 +1,6 @@
 # pylint: disable=missing-function-docstring,missing-class-docstring
+from typing import cast
```

Comment:
The type checker should pass with this change 
```suggestion
from __future__ import annotations

from typing import cast
```